client/src/__tests__/signUp.test.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event'
+import { navigate } from '@reach/router';
+import '@testing-library/jest-dom/extend-expect'
+import SignUp, {localStorage} from '../pages/SignUp';
```

Comment:
HMm there is a mock for localStorage npm package that i've included in one of my changes. If you want you coudl look at updating this test to use that instead (but will also be fine without it)

Reply:
Just seen your test for this in the login test. Will implement it here 🤙